fastos/src/vespa/fastos/unix_file.cpp

```diff
@@ -58,6 +72,12 @@ void FastOS_UNIX_File::ReadBuf(void *buffer, size_t length,
     ssize_t readResult;
 
     readResult = pread(_filedes, buffer, length, readOffset);
+    if (readResult < 0 && _failedHandler != nullptr) {
+        int error = errno;
+        const char *fileName = GetFileName();
+        _failedHandler("read", fileName, error, readOffset, length, readResult);
```

Comment:
should it be "pread" ?

Reply:
This reflects how it is done on linux and what the ioerrorhandler unit test expects.
The error reporting does not differentiate between pread and read, nor between pwrite and write.